tools/copydlldeps.md

```diff
@@ -45,6 +45,24 @@ if [ ! $( echo $compiler | grep -q "shared" ) ]; then
 fi
 ```
 
+Test & example
+==============
+
+Let us compile password manager [KeePassX](https://www.keepassx.org) in
+shared mode (i686-w64-mingw32.shared) and gather all its DLL dependencies.
+
+```bash
+$ make keepassx MXE_PLUGIN_DIRS=plugins/apps \
+    MXE_TARGETS='i686-w64-mingw32.shared'
+$ ./tools/copydlldeps.sh --copy --destdir keepassx-bundle \
+    --infile usr/i686-w64-mingw32.shared/bin/KeePassX.exe \
+    --recursivesrcdir usr/i686-w64-mingw32.shared/
+$ cp -r usr/i686-w64-mingw32.shared/bin/{KeePassX.exe,share} keepassx-bundle
```

Comment:
I think, `copydlldepsexample.sh` is too long to serve as an example. Example should fit in my small 24x80 terminal :). Moreover it is not only an example of using `copydlldeps.sh`, but also of building and deploying. 

Nevertheless it would be useful as an introduction for newcomers, but not in a form of a shell script doing everything (and including "Do NOT edit below this line"), but in form of article with description of each step supplemented with commands. Try to avoid using variables in examples to make it easier to read by humans. Please also rename COMPILER to TARGET, because it is called TARGET in the rest of MXE code. I think, it can be added to `index.html`.